### PR TITLE
Purge cloudflare cache for index pages and homepage

### PR DESCRIPTION
**Pull request by tombola**

### CODE REVIEW

**Comment by richbrennan** [comment]
Looks good, but CI is failing

File: tbx/core/models.py [hunk 1]
```diff
@@ -729,6 +734,18 @@ def __str__(self):
...
+def purge_parent_index(index_page_model, child_page):
+    # Purge FE cache for indexes containing specified child page
+    batch = PurgeBatch()
+    batch.add_pages(index_page_model.objects.live().ancestor_of(child_page))
+
+
+def purge_homepage():
+    # Purge FE cache for homepage
+    batch = PurgeBatch()
+    batch.add_page(HomePage.objects.first())
```
Comment (nealtodd) [inline tbx/core/models.py]: Shouldn't this (and `batch` in `purge_parent_index` call `batch.purge()` after adding the pages to actually do the purge?

Comment (tombola) [reply tbx/core/models.py]: absolutely right, not sure how i missed that

**Comment by zerolab** [comment]
@kaedroho will this be needed with the rebuild?

**Comment by zerolab** [comment]
Thank you for your work on this @tombola. Closing as no longer relevant with the new headless setup

**Pull request closed without merging**